docs/features/private-wlan.rst

```diff
@@ -4,6 +4,13 @@ Private WLAN
 It is possible to set up a private WLAN that bridges the WAN port and is separated from the mesh network.
 Please note that you should not enable ``mesh_on_wan`` simultaneously.
 
+The private WiFi is by default encrypted via WPA2. On devices with enough flash and a supported radio,
+WPA3 or WPA2/WPA3 mixed-mode can be used instead of WPA2.
+
+Additionally, IEEE 802.11w mangagement frame protection should be enforced for WPA2/WPA3, however this
+can lead to connectivity problems for older clients. In this case, mangagement frame protection can be
+set to optional or disabled completely using the config-mode. 
```

Comment:
We usually refer to the web UI as "configuration wizard" or "advanced settings" instead of using the more technical term "config-mode" in the docs.